common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -239,11 +239,6 @@ div.problem {
   input[type="checkbox"] {
     @include margin(($baseline/4) ($baseline/2) ($baseline/4) ($baseline/4));
   }
-
-  text {
```

Comment:
@cahrens. SVG can have text styles in there. Is there a problem type that has a svg?

I made an example on JSFiddle

[https://jsfiddle.net/hxj24ys1/](https://jsfiddle.net/hxj24ys1/)